apps/src/sites/code.org/pages/public/teacher-dashboard/index.js

```diff
@@ -249,6 +255,8 @@ function main() {
           pairingAllowed: s.pairing_allowed,
           numStudents: s.students.length,
           code: s.code,
+          course_id: s.course_id,
+          script_id: s.script ? s.script.id : null,
```

Comment:
That was the reasoning, but I'm changing it for other fields (loginType, stageExtras, etc.) and should probably change it here too.